README.md

```diff
@@ -44,20 +44,20 @@ For each one, type your answer, enter a single number (or just hit return) to ch
 This is the structure cookiecutter will create:
 ```
 my-awesome-software
-	├── LICENSE
-	├── MANIFEST.in
-	├── README.md
-	├── my_awesome_software
-	│   ├── __init__.py
-	├── pyproject.toml
-	├── tests
-    │   ├── __init__.py
-    │   ├── test_integration
-    │   │   └── __init__.py
-    │   └── test_unit
-    │       ├── __init__.py
-    │       └── test_placeholder.py
-	└── tox.ini
+	LICENSE
```

Comment:
It seems a lot harder to me to parse this directory structure without the lines.